server/src/utility/scheduler.js

```diff
@@ -0,0 +1,38 @@
+const schedule = require('node-schedule');
+const sendEmail = require('./sendEmail');
+
+const scheduleEmailNotification = (date, data) => {
+	const { jobName, to, subject, message } = data;
+	schedule.scheduleJob(jobName, date, () => {
+		sendEmail(to, subject, message);
+	});
+};
+
+const removeNotification = data => {
+	const { substring } = data;
+	for (const prop in schedule.scheduledJobs) {
+		if (prop.includes(substring)) {
+			schedule.scheduledJobs[prop].cancel();
+		}
+	}
+};
+
+const rescheduleNotification = (date, data) => {
+	const { prefix } = data;
```

Comment:
I am not sure and a bit skeptical about this prefix, substring thing. Can we figure out a better way?

Reply:
Yes, I'll look for other things.